metadata-ingestion/tests/integration/looker/golden_test_allow_ingest.json

```diff
@@ -497,6 +497,43 @@
         "lastRunId": "no-run-id-provided"
     }
 },
+{
+    "entityType": "platformResource",
+    "entityUrn": "urn:li:platformResource:2c0d2b186264ca5f2532539b69f1559e",
+    "changeType": "UPSERT",
+    "aspectName": "dataPlatformInstance",
+    "aspect": {
+        "json": {
+            "platform": "urn:li:dataPlatform:looker"
+        }
+    },
+    "systemMetadata": {
+        "lastObserved": 1586847600000,
+        "runId": "looker-test",
+        "lastRunId": "no-run-id-provided"
+    }
+},
+{
+    "entityType": "platformResource",
+    "entityUrn": "urn:li:platformResource:2c0d2b186264ca5f2532539b69f1559e",
+    "changeType": "UPSERT",
+    "aspectName": "platformResourceInfo",
+    "aspect": {
+        "json": {
+            "resourceType": "USER_ID_MAPPING",
+            "primaryKey": "",
+            "value": {
+                "blob": "{}",
```

Comment:
In which case would this be empty in real world ? Ideally, we should always receive mappings here.

Reply:
hi @mayurinehate 
In real-world scenarios, mappings are always provided, and I have verified this. However, in the test cases, we don't have user data, which is why the mappings appear empty.